doc/source/migration-guide.rst

```diff
@@ -157,7 +157,7 @@ Your package will most likely have failed pre-commit hooks. We will manually fix
 
 #. For each `flake8` branch, create a PR request to ``package``. Since you are fixing flake8 errors, the commit message can be ``skpkg: Fix flake8 <readable-error-type> errors`` and the pull request title can be ``skpkg: Fix flake8 <readable-error-type> errors``.
 
-#. For each PR, either the project owner or the maintainer will review the PR and merge it to ``package``. If you are the project owner, you can merge the PR yourself.
+#. For each PR, either the project maintainer or the maintainer will review the PR and merge it to ``package``. If you are the project maintainer, you can merge the PR yourself.
```

Comment:
maintainer is repeated here.  Only need it once?  Or in this case, leave owner here as it was before?  It is probably more appropriate that it is owner here.

Reply:
Yes. Thanks for catching this. Should've double checked. 

Changed.

Reply:
done 

```
#. For each PR, the project owner will review the PR and merge it to ``package``.
If you are the project owner, you can merge the PR yourself.
```